frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -153,6 +153,7 @@ end
 function ReaderRolling:onReaderReady()
     self:setupTouchZones()
     self.setupXpointer()
+    self.old_page = 0
```

Comment:
Hm, apparently that caused this problem: `frontend/apps/reader/modules/readerrolling.lua:268: attempt to perform arithmetic on field 'old_page' (a nil value)`

I guess line 267 could be changed to `if self.view.view_mode == "page" and self.old_page then` except I'm not really sure what the else condition would do there? (Also, shouldn't it also have crashed on trying to divide by 0? :-P)